### ISSUE TITLE
Pokebuddy don't launch

### ISSUE THREAD

**Issue by EpicDreams2** [finndev/PokeBuddy#165]
When i lauche pokeBuddy app its loading but the app don't launch a solution ?

**Comment by Vanethir** [comment]
Did you check our [FAQ](https://github.com/finndev/PokeBuddy/wiki/FAQ)? :)
Might solve your problem. If it's still not launching try to move the PokeBuddy.exe from **Loader** to the parent directory and launch it again.

**Comment by curtiola** [comment]
if it just updates and then nothing happens, it could be that you need to download the newest version

**Comment by EpicDreams2** [comment]
no it snot updationg nothing appears when i lauche the app

**Comment by curtiola** [comment]
strange, try to run as admin? check if a firewall or antivirus is blocking it? best to add it to exceptions even if it's not notifying you that it's being blocked it still might be. also did you check the [FAQ](https://github.com/finndev/PokeBuddy/wiki/FAQ) or try Vanethir's suggestion?

**Comment by EpicDreams2** [comment]
yes i tried , my NetFramework version is 4.6.1 and im on windows 10 if it can help

**Comment by curtiola** [comment]
i'm just a user trying to be helpful, i really don't know much about this stuff or windows 10 lol sorry

**Comment by baigwajahat** [comment]
it is not working in windows 7

**Comment by baigwajahat** [comment]
is ther any way to use this in windows 7

**Comment by curtiola** [comment]
im running it on windows 7, did you try the solutions above or in the [FAQ](https://github.com/finndev/PokeBuddy/wiki/FAQ)?
